src/App.jsx

```diff
@@ -1,12 +1,14 @@
 import { useState } from "react";
 import { Routes, Route } from "react-router-dom";
 import Home from "./Components/Home/Home";
+import Article from "./Components/Article/Article";
```

Comment:
Apologies because I think I already put this comment somewhere but can no longer see it. I'd like you to have a rethink of your file structuring. It makes sense to break it out as you have done, but having something called article inside of something called articles is confusing, because in linguistic terms, it's hard to consider the purpose of the directory if it's got the same name as the file (as in, what else could it possibly contain?). I think it would make more sense to go with Articles (to contain everything that relates to articles at all), and then within there have ArticlesList and SingleArticle.